airgun/views/os.py

```diff
@@ -23,6 +23,16 @@ class OperatingSystemDetailsView(View):
     arch_element = ResourceList(
         parent_entity='OperatingSystem', affected_entity='Architect')
 
+    @View.nested
+    class ptable(View):
+        view_tab = Text("//a[@href='#ptable']")
+        ptable_element = ResourceList(
+            parent_entity='OperatingSystem', affected_entity='Ptable')
```

Comment:
Do not use suffixes denoting the "type" of the object. With widgets it is not necessary.

Reply:
Do you mean `parent_entity='OperatingSystem', affected_entity='Ptable'`. If yes, then I did that not for knowing type of elements, but only to form locators further

Reply:
I mean names of the widgets plopped on views ... no ptable_element, just ptable.